berkeley-mobile/Map/MapMarkerDetailView.swift

```diff
@@ -168,10 +173,16 @@ class MapMarkerDetailView: UIView {
     }
     
     @objc func closeView() {
+        
         marker = nil
-        delegate?.didCloseMarkerDetailView(self)
+        UIView.transition(with: self,
+                          duration: 1.0,
+                                 options: [.transitionFlipFromBottom],
+                                 animations: {
+                                 },
+                                 completion: nil)
     }
-
+    //self.delegate?.didCloseMarkerDetailView(self)
```

Comment:
instead of commenting this out, you should add the animation to these lines: https://github.com/asuc-octo/berkeley-mobile-ios/blob/cae66f01b93753c2133770086fcf909c95f4e3a1/berkeley-mobile/Map/MapMarkerDetailView.swift#L34-L36
this is the code that is run when the detail is dismissed.